### ISSUE TITLE
[FEATURE] Support of intelligent eye sensor enabling/disabling

### ISSUE THREAD

**Issue by nikbyte** [revk/ESP32-Faikin#342]
**Daikin hardware**
FTXA35C2V1B, S21 connector

**Is your feature request related to a problem? Please describe.**
This AC has "intelligent eye" sensor which is not controlled by "sensor" option in faikin.
It would be good to figure out why and fix

**Describe the solution you'd like**
Working "sensor" switch in UI.

**Comment by revk** [comment]
If you find the S21 messages, let me know and I'll make it work.

**Comment by nikbyte** [comment]
I enabled messages dump to mqtt and I see no changes in any of registers when I enable/disable this mode from IR remote. :(

**Comment by nikbyte** [comment]
How else I can debug it?

Does faikin publish into mqtt all S21 messages or only part of them? For example, G9 or GD which I don't see in output.

**Comment by revk** [comment]
You can send any S21 to test.

**Comment by nikbyte** [comment]
Yes, I also wanted to try, but I cannot figure out where and how publish it to mqtt. Could you please provide me an example?

**Comment by nikbyte** [comment]
Just for info.

I tried 0x00 0x00 0x00 0x00 and 0xff 0xff 0xff 0xff into all possible D-registers, it doesn't change value of intelligent eye. :(

(of course, I ensured that I see my packets in /tx queue and I ensured that I can enable/disable another features this way).

I'm out of ideas yet.

**Comment by basak** [comment]
I can confirm this isn't working for me either with the same model (both FTXA25C2V1B and FTXA35C2V1B). As it happens I was also trying to debug this yesterday and saw nothing relevant changing with `"debug": true`.

**Comment by geni0602** [comment]
Hi, 
I can also confirm it. The sensor is not functioning over the webGUI  Daikin FTXM50R2V1B. On the remote there is one button to control confort and sensor mode.
![Viber-Bild_2024-07-17_08-00-40-414](https://github.com/user-attachments/assets/215777a2-0736-4e86-b898-86cc35248f4c)
1. press = comfort mode 
2. press = Sensor mode
3. press comfort + sensor mode 
4. off
 when I press the remote button from comfort to sensor mode I can see this change
<img width="374" alt="Bildschirmfoto 2024-07-17 um 08 12 04" src="https://github.com/user-attachments/assets/01aaa14f-9564-4d0d-bc48-7b2ed6dd5aec">
<img width="365" alt="Bildschirmfoto 2024-07-17 um 08 12 13" src="https://github.com/user-attachments/assets/1fd26552-cd19-4878-aaac-7798b944cdc3">


and this ist the whole massage:
{"G1":"13PA","G2":"=;\u0000\u0080","G3":"0\u00FE\u00FE\u0000","G4":"0\u00B0\u00800","G5":"000\u0080","G6":"0004","G7":"00Q@","G8":"0200","GA":"CA30","GB":"0330","GC":"0031","GG":"0D00","GK":"q{51","GM":"5231","GN":"0000","GP":"2A00","GQ":"BB00","GS":"1000","GT":"4000","SH":"542+","SI":"081+","Sa":"561+","SL":"000","Sd":"000","SN":"150+","SM":"640+","SX":"562+","SD":"000"}

**Comment by geni0602** [comment]
Comfort mode is working properly 
![Screenshot_2024-07-17-08-21-11-51_40deb401b9ffe8e1df2f1cc5ba480b12](https://github.com/user-attachments/assets/6f4e4d08-1e21-422b-b6b9-48797a6e9cd8)

**Comment by revk** [comment]
I think we have `sensor` and `comfort` working now, don't we?

There is more work on S21 being done which may allow more options soon.

**Comment by nikbyte** [comment]
@revk no, for AC with intelligent eye it still doesn't work.

**Comment by revk** [comment]
Is that not what `sensor` mode is, or am I getting confused?

**Comment by nikbyte** [comment]
No, it's some different mode. I'm not sure it's supported in S21, but let's see, maybe we'll find it sometimes.
For me the sensor switch does nothing, just ignored.

**Comment by revk** [comment]
OK then I think we need to wait for more S21 research. It is going well.

**Comment by geni0602** [comment]
I am not at home right know, I'll test it next week

**Comment by Take-Off90** [comment]
I have a Emura3 (FTXJ35A2V1BB) which has also the intelligent eye sensor.
Maybe the follwing information is helping:

When I activate the **Sensor Mode** on Faikin it is reducing the LED light of the A/C.
Disabling the Sensor Mode ist not turning it back on.

The Faikin **LED Control Switch** is doing nothing. 

Sensor mode is different from intelligent eye.
Sensor mode can recognize if there are people in the room and then adjust the settings. (used for energy saving)
The intelligent eye can "see" **where** in the room a person is. (Used to "readjust" the airflow away from the person)

**Comment by revk** [comment]
I think we need to just wait for some of the reverse engineering that is going on. Then this can be adjusted.

**Comment by Sonic-Amiga** [comment]
@nikbyte Hello! Still here ?

We have a python script https://github.com/revk/ESP32-Faikin/blob/main/Tools/Simulators/collect_profile.py can you run it on your A/C ? The script will send some commands via mqtt and gather responses. I need the resulting profile which it dumps. Then i'll put this profile into our A/C simulator and see how original Daikin controller handles it.

**Comment by nikbyte** [comment]
Sent back by email.

**Comment by Sonic-Amiga** [comment]
Cool news:
before falling asleep, i put together a .settings file and run it in the sim. m_dtct=1 !!! My controller recognizes it. By comparing with other profiles i'll be able to figure out settings bits.

Bad news: neither old app nor Onecta support this feature. Without some app, that controls it, we can't figure out control commands.

**Comment by MartB** [comment]
@Sonic-Amiga I can toggle the intelligent eye from my onecta app on the original wifi controller on my new Perfera unit. Can i help somehow? I can confirm that the sensor mode is not doing the same thing on my controller too.

**Comment by Sonic-Amiga** [comment]
Hello! What kind of wifi adapter does it natively use? Would be very nice to get high-quality pcb photos

**Comment by Sonic-Amiga** [comment]
Hm, so my C41 should indeed talk to your aircon. Okay, thank you for the profile, I'll try pushing it for some more.

**Comment by Sonic-Amiga** [comment]
@MartB  Hi! Keep fighting; and even with your profile it doesn't want to work. After i disconnect it from (simulated) X50 aircon (which works fine) and connect it to (simulated) FTXM35A5V1B with your profile; the adapted disconnects from cloud and sits in "config" mode, responding this to /config/adapter/product_info:
```
{"rsc":2000,"model_name":"","serial_number":"","region":"eu","device_type":1,"device_kind":"NA"}
```
 I noticed that the adapted also queries for Rz52 and Rz72 commands. Responses from my aircons (both) are '40' and '23' correspondingly. I programmed these, result is the same.
 In order to complete this research, could you tell (using Faikin) what are responses to these two commands from your A/C ? Maybe it tells something different, and the controller does some identification based on these values.

**Comment by Sonic-Amiga** [comment]
> I can toggle the intelligent eye from my onecta app on the original wifi controller on my new Perfera unit. Can i help somehow?

 Unfortunately we seem to be stuck with emulating an S21 A/C with this controller; but it's possible to intercept raw traffic.
 I'm not sure how to use Faikin's snoop mode; never tried/tested it; i instead just connected a PC via a usb-serial adaptor in parallel to one of serial port lines, fired up RealTerm (https://sourceforge.net/projects/realterm/) and simply captured hex dump.
@revk Can you help with snoop mode maybe ?

**Comment by Sonic-Amiga** [comment]
Hello everyone! Some news, bad and good.
Bad: I still cannot befriend BRP069C41 with S21. It just refuses to talk to cloud and sits in "unknown device connected" state.
Good: Thanks to https://github.com/revk/ESP32-Faikin/discussions/220#discussioncomment-11312795 i've downloaded Thai app ("Daikin mobile controller") and reverse engineered it a bit. It mentions at least 3.40 protocol, so i expect BRP072C42 to know some extra features. And from common/basic_info i see HW platform is the same, so i should be able to upgrade my BRP069B41 to 072C42 and get the new functionality

The app refuses to find my BRP069, so i need some tricks and some time. Perhaps i'll write my own firmware download tool.

**Comment by Sonic-Amiga** [comment]
All fixed, all tricks learned. Now i have an upgraded controller, fully recognized by the app. Let's see what new stuff it gives us.

**Comment by MartB** [comment]
Sorry was not really paying attention, is there anything you still require from me?

**Comment by Sonic-Amiga** [comment]
Hi! No, nothing to ask you for. I still cannot replicate your setup, i.e. mt BRP969C41just refuses to connect to the cloud and do anything if i use S21 protocol. It only fully works with X50.
 I am currently trying a different way. Also not doing a whole lot, no time, sorry.
